app/views/comments/index.json.jbuilder

```diff
@@ -1,6 +1,11 @@
 # frozen_string_literal: true
 
 json.array!(@comments) do |comment|
-  json.extract! comment, :id, :user_id, :file_id, :row, :column, :text, :username, :date, :updated, :editable
+  json.extract! comment, :id, :user_id, :file_id, :row, :column, :text
```

Comment:
Please allow me this small refactor with this PR. I think this view-related field should be set in the template and not in the Controller and Model.